src/main/target/SPEEDYBEEF7V3/target.h

```diff
@@ -148,7 +148,8 @@
 
 // ********** Optiical Flow adn Lidar **************
 #define USE_RANGEFINDER
-#define USE_RANGEFINDER_MSP
+//#define USE_RANGEFINDER_MSP
```

Comment:
You don't need to comment out USE_RANGEFINDER_MSP.

That just enables the MSP rangefinder driver.

Unless you are running out of space on the f7 target, in which case this may need to be a new or custom target.

Reply:
yes, please leave this enabled